waku/v2/protocol/rln/group_manager/dynamic/dynamic.go

```diff
@@ -223,6 +227,8 @@ func (gm *DynamicGroupManager) InsertMembers(toInsert *om.OrderedMap) error {
 		}
 		gm.metrics.RecordMembershipInsertionDuration(time.Since(start))
 
+		gm.metrics.RecordRegisteredMembership(startIndex + uint(len(idCommitments)))
+
 		_, err = gm.rootTracker.UpdateLatestRoot(pair.Key.(uint64))
 		if err != nil {
 			return err
```

Comment:
Should this become a critical error that we are not able to updateLatestRoot? 
Eventhough a new member is inserted, if the root is not updated then until another event occurs, the new member will not be valid(which would be an issue) in case no events are noticed for a long time.

Should we need to reload rln or retry in this case?

Reply:
Tracking work for this here - https://github.com/waku-org/go-waku/issues/738
I think we should panic, because the merkle tree will be inconsistent